src/Plate/index.stories.tsx

```diff
@@ -57,6 +58,19 @@ const Template: Story<Partial<PlateProps>> = function Template(args) {
   return <Plate data={null} {...args} />;
 };
 
+const DraggablePlateTemplate: Story<Partial<PlateProps>> =
+  function DraggablePlateTemplate(args) {
+    return (
+      <Plate
+        data={null}
+        {...args}
+        onDragEnd={
+          action('onDragEnd') // dataLocation: `const sourceData = e.active.data.current; const targetData = e.over?.data.current;`
```

Comment:
?

Reply:
A description of how a developer can get the data object of the source (draggable) and target (droppable).